MonkeyChat/NSNotificationCenterKeys.swift

```diff
@@ -0,0 +1,17 @@
+//
+//  NSNotificationCenterKeys.swift
+//  MonkeyChat
+//
+//  Created by Erika Perugachi on 10/14/16.
+//  Copyright © 2016 Criptext. All rights reserved.
+//
+
+import Foundation
+
+struct MonkeyChatNotification {
```

Comment:
let's define this struct like Alamofire does it: (https://github.com/Alamofire/Alamofire/blob/master/Source/Notifications.swift)